core/src/mindustry/net/Administration.java

```diff
@@ -302,11 +302,13 @@ public boolean unbanPlayerID(String id){
     public boolean adminPlayer(String id, String usid){
         PlayerInfo info = getCreateInfo(id);
 
+        if(info.admin) return true;
```

Comment:
Might not want to return early here, so the usid is updated.

Reply:
ah true